pkg/volume/csi/csi_test.go

```diff
@@ -47,6 +48,7 @@ import (
 // based on operations from the volume manager/reconciler/operation executor
 func TestCSI_VolumeAll(t *testing.T) {
 	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.CSIInlineVolume, true)()
+	defaultFSGroupPolicy := storagev1.ReadWriteOnceWithFSTypeFSGroupPolicy
```

Comment:
do we need to test with and without the feature gate on?

Reply:
I'm not certain if testing with the feature gate off would be needed, as we're enabling it by default henceforth?

Reply:
Even though it's on by default, it's possible that someone can still turn the feature off.

Reply:
I've included a test (that passes with the default value) to disable the feature gate. Since it's on by default, the rest of the tests should run with it enabled.